helm/bailo/values.yaml

```diff
@@ -22,12 +26,12 @@ autoscaling:
 podSecurityContext: {}
   # fsGroup: 2000
 securityContext:
-  capabilities:
-    drop:
-      - ALL
-  readOnlyRootFilesystem: true
+  # capabilities:
+  #   drop:
+  #     - ALL
```

Comment:
All permissions should be dropped by default and only the required capabilities added back in via `add` (at the same level of `drop`).